assets/src/edit-story/elements/image/display.js

```diff
@@ -100,6 +100,7 @@ function ImageDisplay({ element, box }) {
         src={src}
         srcSet={srcSet}
         alt={resource.alt}
+        data-testid="imageElement"
```

Comment:
As mentioned below, there's probably no need for this as we can re-use the existing `frameElement` test ID